package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "simple-time-tracker",
-    "version": "1.0.3",
+    "version": "1.1.0",
```

Comment:
Please don't increase the version number as part of this PR - it only gets increased when the tag is actually created :)